tests/test_docker.sh

```diff
@@ -7,10 +7,14 @@
 set -ex
 
 # run the model service's tests
-docker-compose -f local.yml run prediction pytest -rsx
+# Coverage should ignore pip packets, files including tests and pytest
+docker-compose -f local.yml run prediction coverage run --branch --omit=/usr/local/lib/python3.6/dist-packages/*,src/tests/*,/usr/local/bin/pytest /usr/local/bin/pytest -rsx
+docker-compose -f local.yml run prediction coverage html
```

Comment:
IIRC, this generates a directory `htmlcov/` but does not do anything else. We probably want `coverage report`, right?

Reply:
Yes, `coverage html` generates this directory which, when being served, offers an interactive report like [here](https://wgierke.github.io/stuff/prediction/index.html) which is IMO easier to navigate and understand. However, if it's difficult to persist the whole directory by Travis, it should also be fine to just pass the `report` command instead, which creates an output similar to:
```
Name                      Stmts   Miss  Cover   Missing
-------------------------------------------------------
my_program.py                20      4    80%   33-35, 39
```